include/print.h

```diff
@@ -0,0 +1,43 @@
+#ifndef __CHERI_EXAMPLE_PRINT_H
+#define __CHERI_EXAMPLE_PRINT_H
+
+#include <assert.h>
+#include <cheri.h>
+#include <cheri/cheri.h>
+#include <cheri/cheric.h>
+#include <machine/sysarch.h>
+#include <stdbool.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <sys/sysctl.h>
+#include <sys/types.h>
+
+//
+#define cheri_tag_get(x) __builtin_cheri_tag_get(x)
+#define cheri_address_get(x) __builtin_cheri_address_get(x)
+#define cheri_is_sealed(x) __builtin_cheri_sealed_get(x)
+#define cheri_flags_get(x) __builtin_cheri_flags_get(x)
+
```

Comment:
Are these not defined in `cheriintrin.h`? Any reason we're not using that? I thought `cheric.h` was deprecated.

Reply:
cherriintrin.h shows macros, but does not include all necessary things that I need + I had some conflicts in func signature, which I couldn't be bothered to resolve. So I disabled including cheriintrin.h for now.

Reply:
I always get confused by `cheric.h` and `cheriintrin.h` -- but if `cheriintrin.h` is the right one, we should convert the PR to use that.